ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/hosts.yml

```diff
@@ -1,14 +1,6 @@
 ---
 all:
   children:
-    IGNORE_IN_PYTEST:
-      children:
-        CUSTOM_TEMPLATES_TESTS:
-        CUSTOM_PYTHON_MODULES_TESTS:
```

Comment:
Let's see how we can get this one to work in pytest

Reply:
done